### PR TITLE
completeConfig: remove storage, storageErr state from config

### PR DESCRIPTION
**Pull request by sttts**
Cleanup in preparation of token review support.

### CODE REVIEW

File: pkg/oauth/apiserver/apiserver.go [hunk 1]
```diff
@@ -86,14 +82,6 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 	return s, nil
 }
 
-func (c *completedConfig) V1RESTStorage() (map[string]rest.Storage, error) {
```
Comment (stlaz) [inline pkg/oauth/apiserver/apiserver.go:89]: Since we're getting rid of it (yay \o/), please also remove the same code in the user apiserver

**Review by stlaz — CHANGES_REQUESTED** [review]

File: pkg/oauth/apiserver/apiserver.go (hunk 1 above)
Comment (sttts) [reply pkg/oauth/apiserver/apiserver.go:89]: done

**Pull request merged**